src/shogun/CMakeLists.txt

```diff
@@ -360,7 +361,8 @@ SHOGUN_DEPENDENCIES(
 SHOGUN_DEPENDENCIES(
   LIBRARY LibLZMA
   SCOPE PRIVATE
-  CONFIG_FLAG USE_LZMA)
+  CONFIG_FLAG USE_LZMA
+  GPL_ONLY)
```

Comment:
because liblzma is gpl?

Reply:
it a actually is PD, I got this one wrong